loopchain/baseservice/slot_timer.py

```diff
@@ -19,12 +19,13 @@
 class SlotTimer:
     """Slot Timer"""
 
-    def __init__(self, timer_key, duration, timer_service: TimerService, callback):
+    def __init__(self, timer_key, duration, timer_service: TimerService, callback, callback_lock):
```

Comment:
I think this lock isn't needed if you use SlotTimer only consensus because await does not exist in consensus(). So consensus() doesn't call async.
If you consider to use await in callback, this lock is needed